test/Project.toml

```diff
@@ -1,5 +1,6 @@
 [deps]
 Downloads = "f43a241f-c20a-4ad4-852c-f6b1247861c6"
+ONNX = "d0dd6a25-fac6-55c0-abf7-829e0c774d20"
```

Comment:
Hmm, last time I checked, it was not needed to add the package itself to the `test/Project.toml`. Do you encounter an error when running `] test` without it?

Reply:
Ah, didnt catch that. Might be a result of me using `] dev ONNX` to pull down my fork in .julia/dev